typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/helpers/findItemByName.ts

```diff
@@ -24,6 +24,5 @@ import { WorkflowItemDescriptor } from "../../../../decorators";
  * @returns The index of the item with the given name, or 0 if not found
  */
 export function findItemByName(items: WorkflowItemDescriptor[], name: string): number {
-	const index = items.findIndex(item => item.name === name);
-	return index === -1 ? 0 : index + 1;;
+	return 1 + items.findIndex(item => item.name === name);
```

Comment:
Haha, what a cool idea! `1 + (-1) == 0`

Reply:
I believe it had an issue initially - it would have returned 1 for index === -1 (+ 1 is applied after the ternary is evaluated)

Reply:
Nope, seems fine
<img width="348" alt="image" src="https://github.com/user-attachments/assets/4dc48cc7-e4f7-44a4-8d9f-73b1e041ddab">